screensy-website/translations/ja.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE htm/>
```

Comment:
Is this correct? Shouldn't this be `<!DOCTYPE html>`, like the first line of the other translations?